operators/index.adoc

```diff
@@ -8,12 +8,16 @@ toc::[]
 
 include::modules/operators-overview.adoc[leveloffset=+1]
 
-With Operators, you can create applications to monitor the running services in the cluster. Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as autoscaling up and down and creating backups. All these activities are in a piece of software running inside your cluster.
+Operators are designed specifically for Kubernetes-native applications to implement and automate common Day 1 operations on a {product-title} cluster, such as installation and configuration. Operators can also automate Day 2 operations, such as autoscaling up and down and creating backups. All these activities are in a piece of software running inside your cluster.
 
-[id="operators-overview-developer-tasks"]
+Operator Lifecycle Manager (OLM) and OperatorHub are default components in {product-title} that help manage add-on Operators. Together they provide the system for discovering, installing, and managing optional Operators available on the cluster, also known as _OLM-based Operators_.
+
+In {product-title}, all cluster functions are divided into a series of default _cluster Operators_. Cluster Operators are a different type of Operator that manage a particular area of cluster functionality, such as cluster-wide application logging, management of the Kubernetes control plane, or the machine provisioning system. Cluster Operators are installed by default, unless otherwise noted, and are managed by the Cluster Version Operator (CVO) and not by OLM.
```

Comment:
Hmm so again, I am confused about this content and where it belongs. With bullet #1? 

Thinking you might want to refactor this section with the new content and make subheads for OLM-managed Operators and CVO-managed Operators (or by function, Cluster Operators and Optional Operators).